ElectronClient/app/gui/MainScreen.jsx

```diff
@@ -380,6 +388,12 @@ class MainScreenComponent extends React.Component {
 			verticalAlign: 'top',
 		};
 
+		if (isNoteListVisible === false) {
+			this.styles_.noteList.width = 0;
+			this.styles_.noteList.display = 'none';
```

Comment:
Any reason we need both these lines? Shouldn't `display = 'none'` be enough?

Reply:
I don't know why, but if you remove `this.styles_.noteList.width = 0;` you break the sidebar.
It no longer works.

Reply:
@tbergeron do you have an explanation?

Reply:
@laurent22 You are absolutely right! I based myself on the code for sidebar here https://github.com/laurent22/joplin/pull/1991/files/182e17fd7ec29a602ed2db56374ede2f14802017#diff-60bbc35164f34c77abfc1d4f6a0eb1b8R380 but I just tested without the `width = 0` and it was fine so I removed it. Sidebar breaks without it but for some reason the note list doesn't.